packages/node/examples/express/package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "airbrake-example",
   "dependencies": {
-    "@airbrake/node": "^1.0.0-beta.1",
-    "express": "^4.14.1",
-    "request": "^2.81.0"
+    "@airbrake/node": "^1.0.6",
+    "express": "^4.17.1",
+    "pg": "^8.0.0"
```

Comment:
Is `pg` relevant to the example?

Reply:
Yep, it should have been included here for quite some time. In the example app.js, the package is required here:
https://github.com/airbrake/airbrake-js/blob/2f53efc277dbb8f73c7e962084ba995896374f19/packages/node/examples/express/app.js#L2

and called here:
https://github.com/airbrake/airbrake-js/blob/2f53efc277dbb8f73c7e962084ba995896374f19/packages/node/examples/express/app.js#L16